test/src/requestinterception-experimental.spec.ts

```diff
@@ -178,7 +178,7 @@ describe('cooperative request interception', function () {
       page.on('request', request => {
         const headers = Object.assign({}, request.headers(), {
           foo: 'bar',
-          origin: undefined, // remove "origin" header
+          accept: undefined, // remove "accept" header
```

Comment:
The `origin` header was not being set for the request.
`accept` should be there in most cases.